Sources/Shared/Extensions/Array+Tailor.swift

```diff
@@ -4,9 +4,9 @@ public extension Array {
 
   /**
    - Parameter name: String
-   - Returns: A mappable object array, otherwise it returns nil
+   - Returns: A mappable object array, otherwise it returns empty array
    */
-  func objects<T : Mappable>(name: String? = nil) -> [T]? {
+  func objects<T : Mappable>(name: String? = nil) -> [T] {
```

Comment:
It could be one more func that throws, but I just don't need it now. And in this func it's actually always an empty array.